test/check_mongo_orchestration_tests.js

```diff
@@ -1,19 +1,14 @@
 'use strict';
 const shell = require('shelljs');
-if (!shell.which('mongo-orchestration')) {
-  //shell.echo('Warning: please install mongo-orchestration.');
-  // shell.echo('Mongo-orchestration not found. Installing mongo-orchestration.');
-  // shell.exec('pip install mongo-orchestration');
+if (!shell.test('-e', 'mongo-orchestration')) {
+  shell.echo('Mongo-orchestration not found. Installing mongo-orchestration.');
   shell.exec('git clone https://github.com/10gen/mongo-orchestration.git');
   shell.cd('mongo-orchestration');
   shell.exec('pip install . --user');
-  shell.exec('ls');
-  shell.cd('mongo_orchestration');
-  shell.exec('ls')
-  shell.cd('configurations')
-  shell.exec('mo servers/clean.json start');
-  //shell.exec('pip install git+https://github.com/10gen/mongo-orchestration.git --user')
+} else {
+  shell.cd('mongo-orchestration');
 }
-shell.echo('Path to Mongo-Orchestration:');
-shell.echo(shell.which('mongo-orchestration'));
-shell.cd(shell.which('mongo-orchestration'));
+shell.cd('mongo_orchestration');
```

Comment:
right before this, we need to do `mongo-orchestration start`, then we don't need to `cd` to configurations. instead, we can do `../scripts/mo configurations/servers/clean.json start`